data/commodities.txt

```diff
@@ -58,6 +100,11 @@ trade
 		"evaporated milk"
 		"fast food supplies"
 		"fertilizer"
+		"fettucini"
+		"flapjacks"
+		"flour"
+		"food"
```

Comment:
Since `food` is a category name, I'd avoid using it as an individual commodity as well.